src/libraries/System.Private.CoreLib/src/ILLink/ILLink.Substitutions.Shared.xml

```diff
@@ -1,5 +1,11 @@
 <linker>
   <assembly fullname="System.Private.CoreLib">
+    <type fullname="System.Diagnostics.Tracing.EventSource" feature="System.Diagnostics.Tracing.EventSource.IsSupported" featurevalue="false">
```

Comment:
Can you please create a tracking issue for the SDK side of this?
* Add the `Trim=true` metadata for the respective `RuntimeHostConfigurationOption` item.
* Possibly introduce a "public" MSBuild property to control this feature switch (which would then map to the `RuntimeHostConfigurationOption`

/cc @sbomer

Reply:
Opened https://github.com/dotnet/sdk/issues/12217